templates/output.html.twig

```diff
@@ -5,13 +5,19 @@
         <div class="form-group">
             <label for="image">{{ msg('image-url') }}</label>
             <input name="image" id="image" class="form-control" type="url" value="{{ app.request.get('image') }}" required />
-            <p class="help-block">{{ msg( 'image-url-help', [image_hosts] ) }}</p>
+            <p class="help-block">{{ msg( 'image-url-help', image_hosts ) }}</p>
```

Comment:
Won't this pass each host as a separate message parameter?

Reply:
Ah, I see why we were calling listToText() in the controller now :) I've made it so it only does that for the /home route. I felt like the API should return an array but I suppose there aren't going to be many clients programmatically relying on that list.